version.txt

```diff
@@ -1 +1 @@
-0.11.0-beta-incubating-SNAPSHOT
+1.0.0-incubating
```

Comment:
I suppose _after_ the 1.0.0 release is under way, this version ought to be something like `1.0.1-incubating`.